### PR TITLE
created profile card

### PR DESCRIPTION
**Pull request by LiorBraginsky**
Апрувьте господа :)

### CODE REVIEW

File: src/components/ProfileCard.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import { Text, Thumbnail } from 'native-base'
+import PropTypes from 'prop-types'
+import Card from './UI/Card'
+import { TextInfo, TextSubtitle, TextTitle } from './Typography/Text'
+import SubTitle from './Typography/SubTitle';
+
+
+
+
+const styles = StyleSheet.create({
```
Comment (olbesp) [inline src/components/ProfileCard.js]: Remove extra whitespace

Comment (olbesp) [inline src/components/ProfileCard.js]: Remove extra whitespace

File: src/components/ProfileCard.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    <TextTitle>{props.name}</TextTitle>
+                    <TextSubtitle>{props.email}</TextSubtitle>
+                    <TextSubtitle>{props.phoneNumber}</TextSubtitle>
+                </View>
+            </View>
+        </View>
+        <View style={styles.horizontalLine}/>
+        <View>
+            <View style={styles.paymentInformation}>
+                <TextInfo>PaymentInformation</TextInfo>
```
Comment (olbesp) [inline src/components/ProfileCard.js]: Put a space between 'Payment Information' words

File: src/components/ProfileCard.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                <TouchableHighlight
+                    style={styles.button}
+                    onPress={this.editPressHandler}
+                    underlayColor={'transparent'}
+                >
+                    <Text style={styles.textColor}>Edit</Text>
+                </TouchableHighlight>
+            </View>
+            <View style={styles.userInfo}>
+                <Thumbnail large source={{uri: props.avatarPhoto}}  style={styles.avatar}/>
```
Comment (olbesp) [inline src/components/ProfileCard.js]: Make an avatar a little bit smaller

screenshot from 2018-06-05 22-56-42

File: src/components/ProfileCard.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    avatarPhoto: PropTypes.string.isRequired,
+};
+ProfileCard.defaultProps = {
+    name: 'Peter Parker',
+    email: 'peter.parker@gmail.com',
+    phoneNumber: '+39459 595 85',
+    avatarPhoto: 'https://facebook.github.io/react-native/docs/assets/favicon.png',
+};
+
+export default ProfileCard;
```
Comment (olbesp) [inline src/components/ProfileCard.js]: Nice and clean code )

**Comment by LiorBraginsky** [comment]
пофиксил

**Pull request merged**